adrian-greksa/wk01/3-wed/arrays/js/arrays.js

```diff
@@ -62,7 +62,28 @@ console.log(multiplyArray([1, 2, 3, 4]));
 // ## Bonus
 
 // 5. Define a function `reverseString` that computes the reversal of a string. For example, reverseString("jag testar") should return the string "ratset gaj".
+function reverseString(str) {
+    return str.split("").reverse().join("");
+}
+reverseString("hello");
+
+
 // 6. Write a function `findLongestWord` that takes an array of words and returns the length of the longest one.
+const findLongestWord = function (x) {
+    let result='';
+    for (let i = 0; i < x.length; i++) {
+        if (x[i].length >= result[0].length) {
+            result[0] = x[i];
+        } else {
+            continue;
+        }
```

Comment:
The loop would have moved on to the next iteration after your `if` statement anyway, theres no need to `continue`.
`continue` is used to skip over code in a loop that you may not want to run.